kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java

```diff
@@ -786,6 +765,23 @@ public void send(String fromStr, String toStr, String subject, String content, S
 		}
 	}
 
+	private InternetAddress[] stringToInternetAddressArray(String addresses) throws AddressException {
```

Comment:
This seems like a good change, all for using the library. Though doesn't InternetAddresses.parse do everything this does already?

http://stackoverflow.com/a/13854096/3708872
http://docs.oracle.com/javaee/5/api/javax/mail/internet/InternetAddress.html#parse(java.lang.String)